keyboards/oktiboard/info.json

```diff
@@ -8,8 +8,5 @@
       "LAYOUT_ortho_4x4": {
         "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}]
       },
```

Comment:
```suggestion
      }
```

Reply:
Thanks sir 🙌

Reply:
The file won't work without this comma removed, because it makes the syntax.

Reply:
Thanks @noroadsleft , I did 👍